### PR TITLE
feat(website): queue transactions drawer

### PR DESCRIPTION
**Pull request by Ignusmart**
This pull request introduces a transaction queue drawer, designed to enhance the user experience by mimicking the functionality of a shopping cart. This feature enables users to queue transactions seamlessly from both the interact tab and the deploy route. The goal is to streamline the process of managing multiple transactions, improving usability and interaction efficiency.

# queueing transactions using the package search input

https://github.com/usecannon/cannon/assets/3952453/544a6f0f-8f28-4433-861a-a1c0a4100d5d

# queueing transactions from the `interact` section

https://github.com/usecannon/cannon/assets/3952453/1bc92405-d995-4632-b673-03b0afe73839

# Disable button if no wallet/safe selected - Prevent queueing txs if there are already queued txs from a different chain

https://github.com/usecannon/cannon/assets/3952453/cc56d4e9-3f8d-4582-9fba-9287c6b9af56

### CODE REVIEW

**Comment by dbeal-eth** [comment]
@Ignusmart love where this UI is going

probably in the near future we are going to have the txns simulation result just be shown in the "cart" area is my guess.

mentioned this in the call as well, but being able to simulate transactions in succession (i.e. what the result of running txn b is after and including the result of the multation being included with txn a). but that can be delayed a bit

File: packages/website/src/features/Packages/FunctionInput.tsx [hunk 1]
```diff
@@ -50,16 +57,25 @@ export const FunctionInput: FC<{
...
+        return (
+          <AddressInput handleUpdate={_handleUpdate} value={initialValue} />
+        );
       case input.type.startsWith('int') || input.type.startsWith('uint'):
-        return <NumberInput handleUpdate={_handleUpdate} />;
+        return (
+          <NumberInput handleUpdate={_handleUpdate} value={initialValue} />
+        );
       case input.type === 'tuple':
+        // TODO: implement value prop for TupleInput
```
Comment (Ignusmart) [inline packages/website/src/features/Packages/FunctionInput.tsx:70]: could this be implemented in another PR? @noahlitvin

**Review by noahlitvin — APPROVED** [review]
